### PR TITLE
Create a CFH message

### PR DESCRIPTION
**Pull request by casperjureskog**
**PT Story:**   https://www.pivotaltracker.com/story/show/143863477

Changes proposed in this PR:
* Added message tab
* Moved tabs to the bottom for android 
* Added controller and view

**What I have learned working on this feature:**
We have learned to use postman 
and that its important to keep track of heruko

screen shot 2017-04-20 at 16 33 56

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
+.controller('MessageController', function($scope, HelpRequest) {
+  $scope.error = null;
+  $scope.help_request = {};
+
+  $scope.createHelpRequest = function() {
+    HelpRequest.save($scope.help_request, function(response){
+      console.log(response);
+      $scope.message = response.message;
+      // Move on to another page or whatever you want to do
```
Comment (tochman) [inline www/js/controllers.js]: So what's happening after the message is sent?

File: www/js/app.js [hunk 2]
```diff
@@ -33,8 +37,17 @@ angular.module('building-blocks', ['ionic', 'building-blocks.controllers', 'buil
         controller: 'HomeController'
       }
     }
-  });
+  })
 
+  .state('tab.message', {
+    url: '/message',
```
Comment (diraulo) [inline www/js/app.js]: why aren't you using `help_request`, IMO it's good to keep some consistency on how you name your resources and associated routes. It helps when reading the code, I can immediately tell what's what.

File: www/js/app.js [hunk 3]
```diff
@@ -33,8 +37,17 @@ angular.module('building-blocks', ['ionic', 'building-blocks.controllers', 'buil
...
+    views: {
+      'tab-message': {
+        templateUrl: 'templates/message/message.html',
```
Comment (diraulo) [inline www/js/app.js]: same here with your views.

File: www/js/controllers.js [hunk 4]
```diff
@@ -1,5 +1,21 @@
 angular.module('building-blocks.controllers', [])
 
 .controller('HomeController', function($scope, News) {
-   $scope.news = News.query();
+  $scope.news = News.query();
+})
+
+.controller('MessageController', function($scope, HelpRequest) {
```
Comment (diraulo) [inline www/js/controllers.js]: Same here, how about `HelpRequestController`?

**Review by diraulo — CHANGES_REQUESTED** [review]
Try to stay consistent in your naming :)
Also when working on UI/Client stuff, always attach a screenshot of your changes to the PR description

File: www/js/controllers.js (hunk 1 above)
Comment (Scharrre97) [reply www/js/controllers.js]: you will get a thank you message, I just updated the screenshot... have a look @tochman

Comment (tochman) [reply www/js/controllers.js]: That's it? No redirect to another page?

Comment (Scharrre97) [reply www/js/controllers.js]: Well.. not at this moment. Stay agile.

File: www/js/controllers.js [hunk 5]
```diff
@@ -1,9 +1,23 @@
...
+  $scope.help_request = {};
+
+
+  $scope.createHelpRequest = function() {
+    HelpRequest.save($scope.help_request, function(response){
+      console.log(response);
+      $scope.message = response.message;
+      // Move on to another page or whatever you want to do
+    }, function(error){
+      // This error handler needs to be refactored before PR. This is cruede
```
Comment (tochman) [inline www/js/controllers.js]: Please note this comment. Act on it or delete it ;-)

File: www/js/controllers.js (hunk 1 above)
Comment (Scharrre97) [reply www/js/controllers.js]: I think that's out of scope for this feature.. we can add a new feature to our PT for that.

Comment (Scharrre97) [reply www/js/controllers.js]: Or what do you think? @tochman

Comment (tochman) [reply www/js/controllers.js]: Agile does not mean bad UX

**Review by tochman — APPROVED** [review]
Okay.

File: www/js/controllers.js (hunk 5 above)
Comment (Scharrre97) [reply www/js/controllers.js]: yes, totally right there. on our way to fix it :)

**Comment by tochman** [comment]
Can you take a look again @diraulo ?

**Review by diraulo — APPROVED** [review]

**Pull request merged**